test/test.py

```diff
@@ -16,6 +16,7 @@
 is_python_3 = (sys.version_info.major == 3)
 is_windows = (os.name == 'nt')
 
+
```

Comment:
to be revisited and rewritten in pytest, if necessary

the same with `test.sh` - it executes a lot of queries, but never asserts the output.